src/radical/pilot/pmgr/launching/default.py

```diff
@@ -1235,32 +1257,37 @@ def _prepare_pilot(self, resource, rcfg, pilot, expand):
 
             if resource not in self._sandboxes:
 
-                for sdist in sdist_paths:
-                    base = os.path.basename(sdist)
-                    ret['ft'].append({'src': sdist,
-                                      'tgt': '%s/%s' % (session_sandbox, base),
-                                      'rem': False})
+                if stage_sdists:
+                    for sdist in sdist_paths:
+                        base = os.path.basename(sdist)
+                        ret['ft'].append({
+                            'src': sdist,
+                            'tgt': '%s/%s' % (session_sandbox, base),
+                            'rem': False
+                        })
 
                 # Copy the bootstrap shell script.
                 bootstrapper_path = os.path.abspath("%s/agent/%s"
                                   % (self._root_dir, BOOTSTRAPPER_0))
                 self._log.debug("use bootstrapper %s", bootstrapper_path)
 
-                ret['ft'].append({'src': bootstrapper_path,
-                                  'tgt': '%s/%s' % (session_sandbox, BOOTSTRAPPER_0),
-                                  'rem': False})
+                ret['ft'].append({
+                    'src': bootstrapper_path,
+                    'tgt': '%s/%s' % (session_sandbox, BOOTSTRAPPER_0),
+                    'rem': False
+                })
 
                 # Some machines cannot run pip due to outdated CA certs.
                 # For those, we also stage an updated certificate bundle
                 # TODO: use booleans all the way?
                 if stage_cacerts:
 
-                    cc_name = 'cacert.pem.gz'
-                    cc_path = os.path.abspath("%s/agent/%s" % (self._root_dir, cc_name))
-                    self._log.debug("use CAs %s", cc_path)
+                    certs = 'cacert.pem.gz'
```

Comment:
new variable name should be updated in L1367 as well

Reply:
Thanks!